src/cc_catalog_airflow/dags/provider_api_scripts/phylopic.py

```diff
@@ -0,0 +1,357 @@
+        endpoint = 'http://phylopic.org/api/a/image/list/{}/{}'.format(
+                  offset, limit)
+
+    if endpoint == '':
+        logger.warning('Unable to parse endpoint: {}, args: {}'.format(
+                       endpoint, args))
+        return [None]
+
+    result = _get_response_json(endpoint=endpoint, retries=2)
+    image_IDs = []
+
+    if result and result.get('success') is True:
+        data = list(result.get('result'))
+
+        if len(data) > 0:
+            for i in range(len(data)):
+                image_IDs.append(data[i].get('uid'))
+
+    if not image_IDs:
+        logger.warning('No content available!')
+        return [None]
+
+    return image_IDs
+
+
+def _get_meta_data(_uuid):
+    logger.info('Processing UUID: {}'.format(_uuid))
+
+    start_time = time.time()
+    base_URL = 'http://phylopic.org'
+    img_URL = ''
+    thumbnail = ''
+    width = ''
+    height = ''
+    foreign_id = ''
+    foreign_url = ''
+    meta_data = {}
+    creator = ''
+    title = ''
+
+    endpoint = "http://phylopic.org/api/a/image/{}?options=credit+" \
+               "licenseURL+pngFiles+submitted+submitter+taxa+canonicalName" \
+               "+string+firstName+lastName".format(_uuid)
+    request = _get_response_json(endpoint=endpoint, retries=2)
+
+    if request and request.get('success') is True:
+        result = request['result']
+
+    license_url = result.get('licenseURL')
+    img_info = _get_image_info(result)
+
+    if img_info is not None:
+        img_URL = img_info[0]
+        width = img_info[1]
+        height = img_info[2]
+        thumbnail = img_info[3]
+
+    else:
+        logger.warning('Image not detected in url: {}/image/{}'.format(
+                       base_URL, _uuid))
+        return None
+
+    foreign_id = img_URL
+    foreign_url = '{}/image/{}'.format(base_URL, _uuid)
+
+    first_name = result.get('submitter', {}).get('firstName')
+    last_name = result.get('submitter', {}).get('lastName')
+    creator = '{} {}'.format(first_name, last_name).strip()
+
+    taxa = result.get('taxa', [])
+    # [0].get('canonicalName', {}).get('string')
+    taxa_list = None
+    if taxa:
+        taxa = list(filter(
+            lambda x: x.get('canonicalName') is not None, taxa))
+        taxa_list = list(
+            map(lambda x: x.get('canonicalName', {}).get('string', ''), taxa))
+
+    if taxa_list:
+        title = taxa_list[0]
+
+        if len(taxa_list) > 1:
+            meta_data['taxa'] = taxa_list
+
+    if result.get('credit'):
+        meta_data['credit_line'] = result.get('credit').strip()
+        meta_data['pub_date'] = result.get('submitted').strip()
+
+    return [
+            foreign_id, foreign_url, img_URL,
+            thumbnail if thumbnail else '\\N',
+            str(width) if width else '\\N',
+            str(height) if height else '\\N',
+            license_url,
+            creator if creator else '\\N',
+            title if title else '\\N',
+            json.dumps(
```

Comment:
Please leave the `meta_data` dict as a dict type.  That way the `ImageStore` class can verify some things about it.